src/test/mir-opt/const_prop/tuple_literal_propagation.rs

```diff
@@ -1,3 +1,4 @@
+// ignore-endian-big
```

Comment:
Instead of ignoring this test we should add tuples and arrays to the "don't verbosely print the constant" list if all tuple fields or the array element type are on the "don't verbosely print constants of this type" list.